cmd/ebrelayer/txs/relayToCosmos.go

```diff
@@ -22,13 +22,12 @@ import (
 	"github.com/cosmos/peggy/x/ethbridge/types"
 )
 
-// RelayEvent : RelayEvent applies validator's signature to an EthBridgeClaim message
+// RelayLockToCosmos : RelayLockToCosmos applies validator's signature to an EthBridgeClaim message
 //		containing information about an event on the Ethereum blockchain before sending
 //		it to the Bridge blockchain. For this relay, the chain id (chainID) and codec
 //		(cdc) of the Bridge blockchain are required.
 //
-func RelayEvent(chainID string, cdc *amino.Codec, validatorAddress sdk.ValAddress, moniker string, passphrase string, claim *types.EthBridgeClaim) error {
-
+func RelayLockToCosmos(chainID string, cdc *amino.Codec, validatorAddress sdk.ValAddress, moniker string, passphrase string, claim *types.EthBridgeClaim) error {
```

Comment:
split function params into multiple lines

Reply:
and use cosmos-sdk/codec instead of tendermint/go-amino

Reply:
https://github.com/cosmos/peggy/pull/77/commits/be4c43930db5f73c577d2a55782bf464405dd549